source/user-manual/capabilities/vulnerability-detection/cpe_helper.rst

```diff
@@ -0,0 +1,557 @@
+.. Copyright (C) 2019 Wazuh, Inc.
+
+.. _vu_cpe_helper:
+
+CPE Helper
+==========
+
+Since Wazuh 3.11.0, Vulnerability Detector integrates with the National Vulnerability Database to analyze vulnerabilities on Windows agents.
+In a future release, this integration will be extended to the rest of systems.
+
+When Syscollector sends the packages and versions of the agents, they are indexed in Wazuh DB and used by Vulnerability Detector directly.
```

Comment:
Don't talk about `Syscollector` here, it is enough to say that the package inventory of agents is stored in databases.